src/docs/reference/modules/Elasticsearch/Elasticsearch-8.x/.env

```diff
@@ -0,0 +1,29 @@
+# Password for the 'elastic' user (at least 6 characters)
+ELASTIC_PASSWORD=password
```

Comment:
The appsettings sample config needs to be changed too.

Reply:
Yeah I'm trying to make it work with the elastic/changeme creds and it doesn't create any index yet. Seems like it is something that doesn't work anymore or I'm missing a config in the appsettings.json file.

Reply:
OK, thanks.

Reply:
Ok, so I think the issue is that I need to get a Fingerprint from Elasticsearch. I'm trying to find how to get it. Doesn't seem so obvious. Not sure if it is worth it at this point. Maybe better disable auth but that's a shame.

Reply:
Ok mets merge your PR. I will keep this one here if I ever find thé trick with auth.

Reply:
OK then!